contents/docs/feature-flags/common-questions.mdx

```diff
@@ -1,90 +1,178 @@
+In our [JavaScript web](/docs/libraries/js) and [React Native](/docs/libraries/react-native) SDKs, you can reduce the cost of feature flags by reducing the number of requests your client makes to fetch feature flag values for your users.
 
----
+1. Using the the [advanced configuration](/docs/libraries/js#advanced-configuration), set `advanced_disable_feature_flags_on_first_load` to `true`. This stops automatically requesting feature flags whenever a page loads. This is useful when you're immediately calling `posthog.identify()` anyway, which requests flags again.
 
-## I care about latency a lot, and 500ms delays are unacceptable on my servers. Can I do something about this?
+2. Using the the [advanced configuration](/docs/libraries/js#advanced-configuration), set `advanced_disable_feature_flags` to `true`. This stops PostHog automatically requesting feature flags. Instead, use [bootstrapping](/docs/feature-flags/bootstrapping) to load flags exactly once.
 
-Yes, use [local evaluation](/docs/feature-flags/bootstrapping-and-local-evaluation#server-side-local-evaluation). This downloads flag definitions on your servers and evaluates them locally.
+### 2. Reduce local evaluation costs
 
----
+If you're using [local evaluation](/docs/feature-flags/local-evaluation), your bill may be high because of too many requests to fetch feature flag definitions. By default, PostHog fetches these every 30 seconds.
 
-## My feature flags are sending a lot of events, how can I manage this?
+Each request charges 10 credits, so assuming your server is constantly running and making 2 requests per minute, this will charge `10 * 2 * 60 * 24 * 30 = 864,000 credits` each month.
 
-Every library has the option to disable sending these events. Just check the relevant docs for the library for the `send_events` parameter in your `posthog.isFeatureEnabled()` or `posthog.getFeatureFlag()` calls.
+You can reduce this by increasing the [feature flag polling interval](/docs/feature-flags/local-evaluation#step-2-initialize-posthog-with-your-personal-api-key) when initializing PostHog. For example, every 5 minutes instead of every 30 seconds.
 
-However, note that this has a few consequences:
+The drawback of this approach is that whenever you make an update to a feature flag using the PostHog UI, it takes 5 minutes (instead of 30) for that change to roll to your server.
 
-1. The usage tab on the flag will stop showing events since we can't track them anymore.
-2. Experiments that depend on trend goals won't work since we use this event to calculate relative exposure. Convert your trend experiments to funnel experiments instead to make this work.
+> **Note:** Do not use local evaluation in an edge / lambda environment, as this initializes a PostHog instance on every call, which can raise your bill drastically. It's best to use regular flag evaluation instead.
 
----
+### 3. Disable sending `Feature flag called` events
```

Comment:
We don't charge for these events anymore, so can we remove this? :)

Reply:
oh, and keep it under the separate question, around sending too many events

Reply:
The "too many events" question has been removed. Any particular reason to include this if it doesn't contribute to billing? I think we should just remove this entirely

Reply:
Fair enough, makes sense 👍

Reply:
this text is still here though, I'm assuming you'll remove it?